packages/cashier/src/components/cashier-locked/__tests__/cashier-locked.spec.tsx

```diff
@@ -261,12 +279,210 @@ describe('<CashierLocked />', () => {
         mockUseCashierLocked.mockReturnValue(true);
 
         render(<CashierLocked />, {
-            wrapper: ({ children }) => <CashierProviders store={mock_root_store}>{children}</CashierProviders>,
+            wrapper: ({ children }) =>
+                wrapWithRouter(<CashierProviders store={mock_root_store}>{children}</CashierProviders>),
         });
 
         expect(screen.getByText(/Your account has not been authenticated./i)).toBeInTheDocument();
     });
 
+    it('should show the proper message if eu client is not fully authenticated and landed in deposit page', () => {
+        const mock_root_store = mockStore({
+            client: {
+                account_status: { cashier_validation: ['ASK_AUTHENTICATE'] },
+                current_currency_type: 'fiat',
+                mt5_login_list: [
+                    {
+                        account_type: 'demo',
+                        sub_account_type: 'financial_stp',
+                    },
+                ],
+                is_eu: true,
+            },
+        });
```

Comment:
is it possible to set these values before the test cases and then modify the variables like the `is_eu` accordingly?